cypress/support/commands.ts

```diff
@@ -0,0 +1,35 @@
+Cypress.Commands.add("login", (email: string, password: string) => {
+  cy.get('input[placeholder="Email"]').type(email);
+  cy.get('input[type="password"]').type(password);
+  cy.get('button[type="submit"]').click();
+});
+
+Cypress.Commands.add(
```

Comment:
@karenn32 Told you about the commands already. Please, do not remove my comments.